sechub-cli/src/daimler.com/sechub/util/filepathmatcher_test.go

```diff
@@ -43,6 +44,18 @@ func Test_When_double_asterisk_on_start_any_path_is_accepted_when_filename_witho
 	AssertFalse(Filepathmatch("/x/y/z/V/a.txt", "**/b.txt"), t)
 }
 
+func Test_When_double_asterisk_on_start_and_filename_with_asterisk(t *testing.T) {
+	AssertFalse(Filepathmatch("/x/y/z/V/a.txtx", "**/*.txt"), t)
+	AssertTrue(Filepathmatch("/x/y/z/V/b.txt", "**/*.txt"), t)
+}
+
+func Test_When_double_asterisk_also_matching_files_in_current_working_directory(t *testing.T) {
```

Comment:
Nice! I was just starting to write a comment to add some tests about pathes but you did it already. Good work 👍